example/pubspec.yaml

```diff
@@ -4,19 +4,19 @@ publish_to: none
 description: A new Flutter project.
 
 environment:
-  sdk: '>=2.14.0 <3.0.0'
-  flutter: '>=2.8.0'
+  sdk: '>=3.4.3 <4.0.0'
+  flutter: '>=3.22.2'
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_google_places_hoc081098:
     path: ../
-  google_api_headers: ^2.0.0
-  uuid: ^3.0.7
+  google_api_headers: ^4.1.2
```

Comment:
I dont thing, we need this package in this example.

Reply:
@devsideal ,
google_api_headers is also used in the example 🙏 
https://github.com/hoc081098/flutter_google_places_hoc081098/blob/df1d04ade493aa1cbbd0a81aa09353d858498764/example/lib/main.dart#L144